kiva/pdfmetrics.py

```diff
@@ -249,7 +254,7 @@ def freeze(self):
         self.frozen = 1
 
     def isEqual(self, other):
-        return ((enc.name == other.name) and (enc.vector == other.vector))
+        return ((self.name == other.name) and (self.vector == other.vector))
```

Comment:
The variable `enc` doesn't exist in this method. It was probably moved around and the value changed but someone didn't bother to change the variable name.

In general this particular module is quite problematic.